src/sbt-test/dependenciesJson/compile/build.sbt

```diff
@@ -0,0 +1,26 @@
+import scala.xml.XML
```

Comment:
(probably not needed here)

Reply:
should I remove it in the xml tests as well? doesn't seem to be used there either.

Reply:
sounds good to me!